spec/collectionspace/tools/data_helpers_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
@mark-cooper Started a place to eventually start moving some of the stuff from xml.rb. Mainly the stuff that prepares data for writing to XML, leaving the code that *writes* XML in xml.rb. I was thinking safe_split and split_mvfs could also be moved here instead of being scattered in different modules? Not at all wedded to this plan, so if you wanted to organize it differently going forward, I'm happy to follow your lead.